pkgs/by-name/ho/honeyfetch/package.nix

```
@@ -0,0 +1,28 @@
+{
+  lib,
+  libdrm,
+  fetchFromGitLab,
+  rustPlatform,
+}:
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "honeyfetch";
+  version = "1.5.0";
+
+  src = fetchFromGitLab {
+    owner = "ahoneybun";
+    repo = "honeyfetch";
+    tag = "v${finalAttrs.version}";
+    sha256 = "sha256-lFp7L3tcqZ1jAL7V7tfUJDPKO2WCvMekUx+p12fNlcM=";
+  };
+
+  cargoHash = "sha256-u+OF4ali7GoHktY8jihgqUQ+4kFuKQNbiaRUdOJrQfA=";
+
+  buildInputs = [libdrm];
+
+  meta = {
+    description = "A classy neofetch but in Rust";
+    homepage = "https://gitlab.com/ahoneybun/honeyfetch";
+    license = lib.licenses.gpl3Plus;
```

```suggestion
    license = lib.licenses.agpl3Plus;
```

It seems clarified AGPL in your repository